VERSION

```diff
@@ -0,0 +1 @@
+1.6.0.dev
```

Comment:
Any reason to do `.dev` here? In theory I agree it sounds good, but we will need to override it for tagged builds. Personally, I would skip it for now so it closely matches what we are already doing. We can consider adding the tag to the version string Envoy side when we do the final version work there?

Reply:
Adding `dev` is for indicating this is not release version, discussing https://github.com/envoyproxy/envoy/pull/2276#discussion_r158929988. (excuse me but it should be `-dev` not `.dev`.)

For tagged builds, my suggested release workflow is: we will rewrite this VERSION file with release versions (like `1.6.0`), then commit and add a git tag. After the release task is done, we will rewrite this file with next development version (like `1.7.0-dev`?).

Currently, I think this workflow is simpler than adding the tag (git tag?) string to the version string in Envoy side when we do the final version work. But since I'm not familier the current release workflow, any comments and suggestions are very welcome. I want to improve the release workflow as well.

Another suggestion is: rename the `VERSION` file to `RELEASE` or `RELEASE_VERSION`. The content should be version number + an indicator. For example, the content will be `1.5.0+` for current version.

Reply:
I agree that it's useful to have a distinction between release version and the HEAD between releases. I think the scheme in this PR could work if, when we tag, we do a commit first to put a non `-dev` version in place, tag, then a following commit. So, it's two extra commits, which doesn't seem too bad if we only release every few months.

Reply:
OK SGTM as long as we understand the multiple commit implications this will have:
1) Commit final version here (or RC if we start doing that)
2) Commit in envoy repo using SHA of commit in this repo.
3) Tag.
4) Commit here again for next dev version.

Reply:
Yeah, it's exactly the one I expected.